README.md

```diff
@@ -34,62 +34,77 @@ push the whole area forward with the machine learning engineering. For this reas
-of the whole system, and uses a plug-in mechanism to support rich functions including: dataset, training, validations
-and deployment.
+### Audience
+
+[Pipcook][] is intended for Web engineers looking to:
+
+- learn what's machine learning.
+- train their models and serve them.
+- optimize own models for better model evaluation results, like higher accuracy for image classification.
+
+> If you are in the above conditions, just try it via [installation guide](INSTALL.md).
+
+### Subprojects
+
+__Pipcook Pipeline__
+
+It's used to represent ML pipelines consisting of Pipcook plugins. This layer ensures the stability and scalability of the whole system and uses a plug-in mechanism to support rich functions including dataset, training, validations, and deployment.
+
+A Pipcook Pipeline is generally composed of lots of plugins. Through different plugins and configurations, the final output to us is an NPM package, which contains the trained model and JavaScript functions that can be used directly.
+
+> Note: In Pipcook, each pipeline has only one role, which is to output the above-trained model you need. That is to say, the last stage of each pipeline must be the output of the trained model, otherwise, this Pipeline is invalid.
 
 __Pipcook Bridge to Python__
 
-For JavaScript engineers, the most difficult part is the lack of a mature machine learning toolset in the ecosystem.
-To this end, we have opened up the interaction between Python and Node.js at the bottom and can easily call some 
-missing APIs.
+For JavaScript engineers, the most difficult part is the lack of a mature machine learning toolset in the ecosystem. In Pipcook, a module called **Boa**, which provides access to Python packages by bridging the interface of [CPython][] using N-API.
+
+With it, developers can use packages such as `numpy`, `scikit-learn`, `jieba`, `tensorflow`, or any other Python ecology in the Node.js runtime through JavaScript.
 
 ## Quick start
 
 ### Setup
 
 Prepare the following on your machine:
 
-| Installer   | Version range |
+| Installer   | Version Range |
 |-------------|---------------|
-| [Node.js][] | >= 12     |
-| [Python][]  | >= 3.6        |
+| [Node.js][] | >= 12         |
 | [npm][]     | >= 6.1        |
 
 Install the command-line tool for managing [Pipcook][] projects:
 
 ```shell
 $ npm install -g @pipcook/pipcook-cli
+$ pipcook init && pipcook daemon start
 ```
 
-Initialize a project:
+Occuring the download problems? We use [tuna](https://mirror.tuna.tsinghua.edu.cn/help/pypi/) mirror to address this issue:
 
 ```sh
-$ mkdir pipcook-example && cd pipcook-example
-$ pipcook init
+$ pipcook init --tuna
 ```
 
-Occuring the download problems? We use [tuna](https://mirror.tuna.tsinghua.edu.cn/help/pypi/) mirror to address this issue:
+Then run a pipeline:
 
-```sh
-$ pipcook init --tuna
+```shell
+$ pipcook run https://raw.githubusercontent.com/alibaba/pipcook/master/example/pipelines/text-bayes-classification.json
```

Comment:
Suggest to give out --verbose version of this command.  Probably also give another command for --tuna args.

Reply:
Should be the `--verbose` option set by default?

Reply:
I think so for new users

Reply:
So shall we keep this here and change it at CLI?

Reply:
Good idea